tasks/webhook

```diff
@@ -171,6 +173,19 @@ class ReleaseHandler(github_handler.GithubHandler):
         publish_to_queue('webhook', json.dumps(body))
         return None
 
+    def handle_issues_event(self, event, request):
+        action = request['action']
+        if event == 'issues' and action not in ['opened', 'synchronize', 'edited']:
```

Comment:
What is "synchronize" for an issue?

Also, isn't the `event == 'issues'` redundant here?

Reply:
wel li want to catch someone unchecking the checkbox; i should probably just see what event that is and catch that :)

Reply:
just realized then if we want to listen to checkboxes, we'd also need to listen to PRs, but a different action.

Reply:
just verified, checkboxes on issues are 'edited'.  On PRs we'll need to listen to the same, but pass it to issue-scan I guess ?

Reply:
Right, whenever the description of a PR changes we need to pipe it through issue-scan.